src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -72,16 +72,31 @@
       az managed-cassandra cluster list
 """
 
-helps['managed-cassandra cluster node-status'] = """
+helps['managed-cassandra cluster backup-list'] = """
```

Comment:
Generally we will design CLI commands like `managed0cassandra cluster backup list`